server/testutil/testgrpc/testgrpc.go

```diff
@@ -0,0 +1,86 @@
+package testgrpc
+
+import (
+	"context"
+	"math/rand"
+	"net"
+	"testing"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"github.com/mwitkow/grpc-proxy/proxy"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
```

Comment:
can you add a comment with an example usage of these new methods?

Reply:
Done